### PR TITLE
feat(query): added EFS Volumes Not Encrypted for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if "encryption" is enabled for EFS Volumes
- **category and severity are provisory**

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/efs_volumes_not_encrypted/query.rego [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+CxPolicy[result] {
+	efs := input.document[i].resource.aws_ecs_task_definition[name].volume
+	volume := efs.efs_volume_configuration
+	not common_lib.valid_key(volume, "transit_encryption")
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_ecs_task_definition[%s].volume.efs_volume_configuration", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "aws_ecs_task_definition.volume.efs_volume_configuration' is defined and not null",
+		"keyActualValue": "aws_ecs_task_definition.volume.efs_volume_configuration' is undefined or null",
```
Comment (felipe-avelar) [inline assets/queries/terraform/aws/efs_volumes_not_encrypted/query.rego:14]: I think `searchLine` is missing

File: assets/queries/terraform/aws/efs_volumes_not_encrypted/query.rego [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	efs := input.document[i].resource.aws_ecs_task_definition[name].volume
+	volume := efs.efs_volume_configuration
+	common_lib.valid_key(volume, "transit_encryption")
+	volume.transit_encryption != "ENABLED"
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_ecs_task_definition[%s].volume.efs_volume_configuration.transit_encryption", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "aws_ecs_task_definition.volume.efs_volume_configuration.transit_encryption' is defined and not null",
+		"keyActualValue": "aws_ecs_task_definition.volume.efs_volume_configuration.transit_encryption' is undefined or null",
```
Comment (felipe-avelar) [inline assets/queries/terraform/aws/efs_volumes_not_encrypted/query.rego:28]: I think `searchLine` is missing

**Review by felipe-avelar — CHANGES_REQUESTED** [review]
I think this could have another positive sample, when `transit_encryption = "DISABLED"`

**Comment by rogeriopeixotocx** [comment]
@tiagocruz20 please check query https://github.com/Checkmarx/kics/blob/master/assets/queries/terraform/aws/ecs_task_definition_volume_not_encrypted/query.rego

**Comment by rogeriopeixotocx** [comment]
@tiagocruz20 please check query https://github.com/Checkmarx/kics/blob/master/assets/queries/terraform/aws/ecs_task_definition_volume_not_encrypted/query.rego

**Pull request closed without merging**